tests/types.py

```diff
@@ -0,0 +1,9 @@
+try:
+    from twisted.internet.testing import MemoryReactor, MemoryReactorClock
+except ImportError:
+    # Twisted 19.7 moved stuff from twisted.test.proto_helpers to
+    # twisted.internet.testing. Let's do the import once rather than doing it
+    # in lots of different files.
+    from twisted.test.proto_helpers import MemoryReactor, MemoryReactorClock
```

Comment:
given it's still available in `twisted.test.proto_helpers` in current twisted, why not just import it from there?

Reply:
I was leaning towards "make it obvious what the deprecated thing is". No strong opinions though, and we have your CI change to detect any twisted trunk breakages. I'll make that change quickly now.